### ISSUE TITLE
Does not work anymore in Safari 9 

### ISSUE THREAD

**Issue by DOFfactory** [canisbos/CustomReader#1]
Hi, unfortunately CustomReader stopped working in Safari 9 (9537.86.1.56.2). Any updates planned?

**Comment by canisbos** [comment]
Stopped working at all?



    _____________________________

**Comment by DOFfactory** [comment]
Yup, loads the default "Reader" not CustomReader. I uninstalled-reinstalled it, but no avail. Still loads the default Reader.

**Comment by canisbos** [comment]
Try resetting to defaults and then change one thing, like text color. See if it takes effect. 






On Mon, Oct 5, 2015 at 8:20 AM -0700, "DOFfactory" <notifications@github.com> wrote:












Yup, loads the default "Reader" not CustomReader. I uninstalled-reinstalled it, but no avail. Still loads the default Reader.

**Comment by DOFfactory** [comment]
Deleted the extension from Library/Safari/Extensions/ folder, Reset Safari (chosen all from the list), started safari, downloaded the extension, installed, still nothing. The extension is loaded in the Preferences, but when I hit Reader button, it is not the extension that is loaded. At least there is no little setting icon next to the little A - big A.

**Comment by ndunand** [comment]
Same here on Safari 9. However, the reader mode is still automatically firing on sites I had set up for Custom Reader to go into read mode automatically. So I guess it is still active somehow, but the original Safari reader mode is kind of overriding its display.

**Comment by DOFfactory** [comment]
Oh, well I have no automatic sites setup.

**Comment by canisbos** [comment]
In Safari 9, CustomReader's settings icon is gone because Reader's own settings icons are not on the page anymore. You have to use the context menu to access settings. 
I have to update the web page to reflect this. 






On Mon, Oct 5, 2015 at 8:38 AM -0700, "DOFfactory" <notifications@github.com> wrote:












Oh, well I have no automatic sites setup.

**Comment by ndunand** [comment]
Right, thanks for the info about the context menu – I had missed that completely.

**Comment by DOFfactory** [comment]
Well, I have to follow up on this issue. The problem is that I have installed CustomReader in Safari 9. Whenever I push the Reader button on a new page, the default Safari reader pops up, no matter what. Then I right clict for the preferences of CustomReader, and I see that the Font and few other settings are removed. When I re-apply them, CustomReader takes control again. However, when I disable and enable the Reader through the button of the search field in Safari, again, the default Safari reader pops up.

**Comment by canisbos** [comment]
Okay,
I'll fix this when I fix the other stuff. 






On Mon, Oct 19, 2015 at 3:14 AM -0700, "DOFfactory" <notifications@github.com> wrote:












Well, I have to follow up on this issue. The problem is that I have installed CustomReader in Safari 9. Whenever I push the Reader button on a new page, the default Safari reader pops up, no matter what. Then I right clict for the preferences of CustomReader, and I see that the Font and few other settings are removed. When I re-apply them, CustomReader takes control again. However, when I disable and enable the Reader through the button of the search field in Safari, again, the default Safari reader pops up.

**Comment by DOFfactory** [comment]
Ok, thanks. ;)

**Comment by DOFfactory** [comment]
Any news on this issue?

**Comment by DOFfactory** [comment]
Bump.
